content/src/service/deployments/DeploymentManager.ts

```diff
@@ -200,8 +217,12 @@ export type DeploymentOptions = {
   sortBy?: DeploymentSorting
   offset?: number
   limit?: number
+  lastId?: string
 }
 
-export type PointerChangesFilters = Pick<DeploymentFilters, 'fromLocalTimestamp' | 'toLocalTimestamp' | 'entityTypes'>
+export type PointerChangesFilters = Pick<
+  DeploymentFilters,
+  'from' | 'to' | 'fromLocalTimestamp' | 'toLocalTimestamp' | 'entityTypes'
```

Comment:
Would it make sense to exclude 'fromLocalTimestamp' & 'toLocalTimestamp'?

Reply:
I didn't want to delete them as they are currently supported on that endpoint. What do you think?

Reply:
I think that you already handled that at the controller level, and you are defining a type used at the service layer. You will never get those parameters here, right?